File: java/array/easy/LargestElementInTheArray.java

public class LargestElementInTheArray{
  public static int largestElement(int[] arr) {
    int max = Integer.MIN_VALUE;
    for (int num : arr) {
      if (num > max) {
        max = num;
      }
    }
    return max;
  }

  public static void main(String[] args) {
    int[] arr1 = {2,5,1,3,0};
    int[] arr2 = {8,10,5,7,9};
    System.out.println(largestElement(arr1));
    System.out.println(largestElement(arr2));
  }
}

File: python/array/easy/largest_element_in_the_array.py
import sys
def largest_element(arr):
  max_element = -sys.maxsize-1
  for num in arr:
    if num > max_element:
      max_element = num
  return max_element
  

if __name__ == "__main__":
  arr1 = [2,5,1,3,0]
  arr2 = [8,10,5,7,9]
  arr3 = [23,21,32,4]
  print(largest_element(arr1))
  print(largest_element(arr2))
  print(largest_element(arr3))